src/ngx_http_lua_socket_tcp.c

```diff
@@ -576,21 +588,37 @@ ngx_http_lua_socket_tcp_connect(lua_State *L)
 
     dd("lua peer connection log: %p", pc->log);
 
-    lua_rawgeti(L, 1, SOCKET_TIMEOUT_INDEX);
-    timeout = (ngx_int_t) lua_tointeger(L, -1);
-    lua_pop(L, 1);
+    lua_rawgeti(L, 1, SOCKET_CONNECT_TIMEOUT_INDEX);
+    lua_rawgeti(L, 1, SOCKET_SEND_TIMEOUT_INDEX);
+    lua_rawgeti(L, 1, SOCKET_READ_TIMEOUT_INDEX);
```

Comment:
@doujiang24 I'm not talking about the metatable, but the object table itself. Yes, I saw you increased the initial size of the metatables but I didn't see changes in the object table. Will you confirm that?